src/pages/home/IncidentCard.tsx

```diff
@@ -58,30 +70,41 @@ type Props = {
   incident: IncidentInterface
 }
 const IncidentCard = (props: Props) => {
+  const [index, setIndex] = useState('')
   const { incident } = props
   const incidentTimestamp =
     incident.updates[incident.updates.length - 1].timestamp
-  const state = getState(incident)
+  const status = getState(incident)
   return (
-    <Card state={state}>
+    <Card state={status}>
       <TitleLink to={`/edit/${incident.id}`}>
         <h1>{incidentTimestamp.toDate().toUTCString()}</h1>
         <h2>{incident.title}</h2>
       </TitleLink>
-      <ServiceList>
-        {incident.services.map(service => (
-          <li key={service}>{service}</li>
-        ))}
-      </ServiceList>
-      <LinkButton
-        to={`/update/${incident.id}`}
-        css="display:inline-block;margin-top:1em;"
+      <ExpandButton
+        onClick={evt => {
+          if (index) setIndex('')
+          else setIndex(incident.id)
+        }}
       >
-        Add an update
-      </LinkButton>
-      {incident.updates.map(update => (
-        <IncidentUpdate key={update.timestamp.seconds} update={update} />
-      ))}
+        +
+      </ExpandButton>
+      <Collapse isOpened={status !== 'stable' || incident.id === index}>
```

Comment:
We can easily create our own `Collapse` element by setting `height: 0` or `height: auto`. Just remember to set the `overflow: hidden`